ci/jenkins.sh

```diff
@@ -7,7 +7,7 @@ set -ex
 
 : ${CCACHE_DIR:=${WORKSPACE}/ccache}
 : ${CCACHE_BASEDIR:=${WORKSPACE}/build}
-: ${CCACHE_HASHDIR:=false}
+: ${CCACHE_HASHDIR:=true}
```

Comment:
Please pull master, this is fixed in commit 801327f516e118f9fa6ee63df5e787fa8c163774.